### PR TITLE
Mm summary module

### PR DESCRIPTION
**Pull request by monicamcgibbon**
Hi Vicky, 

Here's the PR I mentioned in our catch-up that contains more than just the summary tab... as mentioned summary tab formatting is still needed for i.e. the buttons etc which can come back to. It also includes changes to the tab header module (which adds the selected profile/geographies and contains the geography filters). I think the code is now easier to follow than before for this?

There's also a new module Abbie has created, which creates some buttons on the landing page to navigate to different tabs (like what the profile buttons do).

Have also started off a PHS theme off in the global script so people can get an idea of how that will work when we expand on it.

I've added in lots of comments but aware they might only make sense to me so let me know if anything's unclear. Including Abbie as a reviewer too to make sure my comments and code are making sense! 

I realised I can technically progress with the rank tab stuff without this needing to be integrated so no immediate rush to get this all reviewed after all.

### CODE REVIEW

File: shiny_app/scotpho_reduced.png
Comment (vaelliott) [inline shiny_app/scotpho_reduced.png:1]: is this png being used? should it be here or in the www folder?

Comment (vaelliott) [reply shiny_app/scotpho_reduced.png:1]: ah actually its used in the summary table pdf report so maybe its needed in its current location

**Review by vaelliott — APPROVED** [review]
I think the code updates make sense and the comments make sense (although I can see how  we may need to come back and edit some of the comments as the app grows in complexity and modules start calling other modules) .
I don't know why the PDF download won't work for me but does work for you - i'd be interested to know if Abbie and Calli can make it work or if you somehow have the 'golden config?

File: shiny_app/modules/download_data_mod.R
Comment (callid01) [inline shiny_app/modules/download_data_mod.R:1]: The data download button doesn't look the same as the other two (no black outline or shaded when hovered) but sure this isn't a priority!

**Review by callid01 — APPROVED** [review]

**Comment by monicamcgibbon** [comment]
Thanks for reviewing Calli 😊 The PDF download didn’t used to work for me too until deployed so no idea why it’s started working or me now! Also still need to do a lot of styling/formatting so the buttons will all be tidied up at some point, alongside working out what to display in the summary for Scotland.

**Pull request closed without merging**